.github/workflows/sql-test-and-build-workflow.yml

```diff
@@ -19,14 +19,25 @@ on:
 
 jobs:
   build:
+    env:
+      BUILD_ARGS: ${{ matrix.os_build_args }}
     strategy:
+      # Run all jobs
+      fail-fast: false
       matrix:
-        java:
-          - 11
-          - 17
-    runs-on: ubuntu-latest
+        java: [11, 17]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - os: windows-latest
+            os_build_args: -x doctest -x integTest -x jacocoTestReport -x compileJdbc
```

Comment:
Disabled doctest/integTest for now, created an issue to enable them in the future
#921